dumpling/export/dump.go

```diff
@@ -890,6 +896,10 @@ func prepareTableListToDump(tctx *tcontext.Context, conf *Config, db *sql.Conn)
 }
 
 func dumpTableMeta(conf *Config, conn *sql.Conn, db string, table *TableInfo) (TableMeta, error) {
+	if table.Type == TableTypeSequence {
```

Comment:
This will never happen because dumpling doesn't generate TableType through `ParseTableType`.
It's at https://github.com/pingcap/tidb/blob/651e910/dumpling/export/sql.go#L275